### PR TITLE
refactor: removed unwanted stuff and other cleanup

### PR DESCRIPTION
**Pull request by KambojRajan**
1) Most of the refactoring has been completed, but certain areas such as the chat input and message elements require further attention(will be doing soon).
2) A minor design change has been implemented in second stage of seller panel; although it may not enhance the appearance significantly, it is crucial for ensuring proper functionality.
3) auth access shifted from client to server component

### CODE REVIEW

File: app/components/dialog.tsx [hunk 1]
```diff
@@ -39,10 +56,21 @@ function Dialog({
...
+        setTimeout(() => router.back(), 140);
+      }}
+    >
       <Portal forceMount>
         <Overlay asChild>
-          <div className="fixed inset-0 z-50 backdrop-blur-sm" />
+          <div
+            className={cn(
+              "fixed inset-0 z-50 backdrop-blur-sm data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
+              (location === "start" || location === "end") &&
```
Comment (heydoyouknowme0) [inline app/components/dialog.tsx:70]:
```suggestion
              (location =!"center") &&
```

File: app/components/dialog.tsx [hunk 2]
```diff
@@ -39,10 +56,21 @@ function Dialog({
...
+                "fixed inset-0 z-50 bg-black/80  data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
```
Comment (heydoyouknowme0) [inline app/components/dialog.tsx:71]:
```suggestion
                "backdrop-blur-sm":"bg-black/80",
```

File: app/components/dialog.tsx [hunk 3]
```diff
@@ -39,10 +56,21 @@ function Dialog({
...
+      onOpenChange={() => {
+        setTimeout(() => router.back(), 140);
+      }}
+    >
       <Portal forceMount>
         <Overlay asChild>
-          <div className="fixed inset-0 z-50 backdrop-blur-sm" />
+          <div
+            className={cn(
+              "fixed inset-0 z-50 backdrop-blur-sm data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
```
Comment (heydoyouknowme0) [inline app/components/dialog.tsx:69]:
```suggestion
              "fixed inset-0 z-50 data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
```

File: app/components/dialog.tsx [hunk 4]
```diff
@@ -53,9 +81,9 @@ function Dialog({
           >
             <div
               className={cn(
-                "relative max-h-full w-full max-w-full bg-background p-6 shadow-lg sm:p-8",
-                "duration-200 data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95",
-                location === "center" ? "sm:rounded-lg" : "h-full rounded-none",
+                "relative max-h-full w-full max-w-full bg-background shadow-lg",
+                location === "center" && "p-6 sm:p-8",
```
Comment (heydoyouknowme0) [inline app/components/dialog.tsx:85]:
```suggestion
```

File: app/components/dialog.tsx [hunk 5]
```diff
@@ -24,6 +24,23 @@ const dialogLocations = cva(
...
+const contentVariants = cva(
+  "duration-200 data-[state=open]:animate-in data-[state=closed]:animate-out",
+  {
+    variants: {
+      location: {
+        start:
+          "h-full rounded-none ease-in-out data-[state=closed]:slide-out-to-left data-[state=open]:slide-in-from-left",
+        end: "h-full rounded-none ease-in-out data-[state=closed]:slide-out-to-right data-[state=open]:slide-in-from-right",
+        center:
+          "data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 sm:rounded-lg",
```
Comment (heydoyouknowme0) [inline app/components/dialog.tsx:36]:
```suggestion
          "p-6 sm:p-8 data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 sm:rounded-lg",
```

**Comment by KambojRajan** [comment]
anyone tell me ki ye deployment details kese dekhu, access given hai mere ko?

**Comment by heydoyouknowme0** [comment]
Merge ************

**Comment by KambojRajan** [comment]
these are too many changes? argar u wanted all these tho y u not added any review, I requested u ri8?

**Review by geekysilento — APPROVED** [review]

**Pull request merged**